tests/unit/IonBinaryWriterTest.js

```diff
@@ -813,6 +813,22 @@ define([
       }
     }
 
+    errorTest('Cannot pass single string as an annotation.',
+        (writer) => { writer.setAnnotations('taco'), writer.writeInt(5) });
+    errorTest('Cannot pass annotations array without a string.',
+        (writer) => { writer.setAnnotations([5]), writer.writeInt(5) });
+    errorTest('Cannot add int as annotation.',
+          (writer) => { writer.addAnnotation(5), writer.writeInt(5) });
+    errorTest('Cannot add array of chars.',
+          (writer) => { writer.addAnnotation(['t', 'a', 'c', 'o']), writer.writeInt(5) });
+    errorTest('Cannot pass annotations array containing a non string value.',
+        (writer) => { writer.setAnnotations(['a', 5,'t']), writer.writeInt(5) });
+    errorTest('Cannot pass annotations array containing undefined.',
+        (writer) => { writer.setAnnotations([undefined]), writer.writeInt(5) });
+    errorTest('Cannot pass annotations array containing null',
+        (writer) => { writer.setAnnotations([null]), writer.writeInt(5) });
```

Comment:
add a happy case test for `setAnnotations(null)`

Reply:
done